app/models/event.rb

```diff
@@ -1,4 +1,4 @@
 class Event < ApplicationRecord
-  belongs_to :workshop
+  has_one :workshop
```

Comment:
The workshop is the topic that we teach during an event. so a workshop has many events. and an event belongs to a workshop